std_msgs/Header header
float64 z_pos              # z_pos reading
float64 z_pos_variance     # z_pos_variance reading


